internal/access/request.go

```diff
@@ -29,3 +30,10 @@ type Authenticated struct {
 	User         *models.Identity
 	Organization *models.Organization
 }
+
+func (n Authenticated) OrganizationID() uid.ID {
+	if org := n.Organization; org != nil {
+		return org.ID
+	}
+	return 0
```

Comment:
Should we return something that isn't possible (like `-1`) just to be safe?

Reply:
`0` should be just as safe as `-1`, neither should make any rows. I think we can make the `uid.ID.Value` return nil instead of the 0 int, which would help it fail when the column is set to not nil.